### PR TITLE
More info on "remote origin already exists"

### PR DESCRIPTION
**Pull request by moritzebeling**
#### Troubleshooting: Remote origin already exists
* Before adding the new remote, remove the existing one with `git remote remove origin`

Let’s give some information on how you can remove your existing origin before adding a new one. This is exactly what I needed when I was referred to this document, but couldn’t find anything about it.

### Why:

To make the doc more helpful for people running into the "Remote repository already exists" error.

### What's being changed:

I added a small command that lets you remove the existing remote origin
```
git remote remove origin
```

### Check off the following:

- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by Ihsan-K** [comment]
Thanks

**Comment by janiceilene** [comment]
@moritzebeling Thanks so much for opening a PR! I'll get this triaged for review ⚡

File: content/github/getting-started-with-github/getting-started-with-git/managing-remote-repositories.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ $ git remote add origin https://{% data variables.command_line.codeblock %}/octo
 To fix this, you can:
 * Use a different name for the new remote
 * Rename the existing remote repository
-* Delete the existing remote repository
+* Before adding the new remote, remove the existing one with `git remote remove origin`
```
Comment (felicitymay) [inline content/github/getting-started-with-github/getting-started-with-git/managing-remote-repositories.md:59]: Thanks for suggesting this change ✨ 

Currently, this set of bullet points gives a list of ways to fix the problem without explaining how. Your change adds an example of the command for one of the points. I think the best fix is to link to the sections later in this document that explain how to rename and delete a remote. That way, users will have all the information and won't just follow the example command (which will only work if the remote that they want to delete is called `origin`).

```suggestion
* Delete the existing remote repository before you add the new remote. For more information, see "[Removing a remote repository](#removing-a-remote-repository)" below.
```

File: content/github/getting-started-with-github/getting-started-with-git/managing-remote-repositories.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ $ git remote add origin https://{% data variables.command_line.codeblock %}/octo
 To fix this, you can:
 * Use a different name for the new remote
 * Rename the existing remote repository
```
Comment (felicitymay) [inline content/github/getting-started-with-github/getting-started-with-git/managing-remote-repositories.md:58]: I suggest adding a link here too:

```suggestion
* Use a different name for the new remote.
* Rename the existing remote repository before you add the new remote. For more information, see "[Renaming a remote repository](#renaming-a-remote-repository)" below.
```

**Review by felicitymay — COMMENTED** [review]
Thanks for suggesting this change to the docs ✨ 

To ensure that this change is clear to everyone, regardless of their set up and naming conventions, I've recommended an alternative approach.

**Comment by felicitymay** [comment]
You can see the preview of the revised article here: https://docs-7176--patch-1.herokuapp.com/en/github/getting-started-with-github/getting-started-with-git/managing-remote-repositories#adding-a-remote-repository.

**Review by felicitymay — APPROVED** [review]

**Pull request merged**